.circleci/test-api.sh

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env bash
+
+apt-get update
+apt-get install -y jq
+
+make serve-local &
+
+curl --retry-connrefused \
+    --retry 5 \
+    --retry-delay 20 \
+    http://127.0.0.1:5000 --fail
+
+var=$?
+
+if [[ $var -ne 0 ]]
+then
+  echo "Server failed to start."
+  exit $var
+fi
+
+response=$(curl --write-out '%{http_code}' --silent --output /dev/null http://127.0.0.1:5000/spec)
```

Comment:
Performance: why make two requests instead of using this request to check for availability? 🙂

Reply:
The idea as we discussed was to have explicit error messages :)